app/src/main/java/com/io/movio/ui/movielist/adapter/MoviesAdapter.kt

```diff
@@ -33,12 +41,26 @@ class MoviesAdapter(private val listener: ItemOnClickListener)
         notifyDataSetChanged()
     }
 
-    override fun getItemCount(): Int  =  movies.size
+    override fun getItemCount(): Int = movies.size
 
     class MovieViewHolder(val binding: MovieRowBinding) : RecyclerView.ViewHolder(binding.root)
 
     interface ItemOnClickListener {
         fun onItemClick(id: Int)
     }
 
+}
+private fun getGenreToString(genre: List<Int>): String {
```

Comment:
too complicated. Just add the string resource representation in the enum and use it instead. Check next comment